### PR TITLE
Fix for indexes and constraints

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java [hunk 1]
```diff
@@ -74,57 +74,49 @@ public class Allele extends GenomicEntity {
 	@OneToMany(mappedBy = "subject", cascade = CascadeType.ALL)
 	private List<AlleleDiseaseAnnotation> alleleDiseaseAnnotations;
 
-	@IndexedEmbedded(includePaths = { "mutationTypes.curie", "mutationTypes.name", "evidence.curie",
-			"mutationTypes.curie_keyword", "mutationTypes.name_keyword", "evidence.curie_keyword"})
+	@IndexedEmbedded(includePaths = { "mutationTypes.curie", "mutationTypes.name", "evidence.curie", "mutationTypes.curie_keyword", "mutationTypes.name_keyword", "evidence.curie_keyword"})
```
Comment (cmpich) [inline src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java:77]: why not leaving two lines? One line is too long.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java:77]: to me its easier to read when not wrapped... plus I have a big montior

Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java:77]: We have the coding convention: no lines longer than 120 characters.

**Review by cmpich — APPROVED** [review]
Looks good. We need to agree on a coding style for eclipse and IntelliJ so we are not going to overwrite each other's changes.

**Pull request merged**